pkg/networkutils/network.go

```diff
@@ -49,6 +51,19 @@ const (
 	// secondary ENI IP addresses.  If set to "true", the SNAT iptables rule and off-VPC ip rule will not
 	// be installed and will be removed if they are already installed.
 	envExternalSNAT = "AWS_VPC_K8S_CNI_EXTERNALSNAT"
+
+	// envConnmark is the name of the environment variable that overrides the default connection mark, used to
+	// mark traffic coming from the primary ENI so that return traffic can be forced out of the same interface.
+	// Without using a mark, NodePort DNAT and our source-based routing do not work together if the target pod
+	// behind the node port is not on the main ENI.  In that case, the un-DNAT is done after the source-based
+	// routing, resulting in the packet being sent out of the pod's ENI, when the NodePort traffic should be
+	// sent over the main ENI.
+	envConnmark = "AWS_VPC_K8S_CNI_CONNMARK"
```

Comment:
Is there any reason to make this configurable? In what usecases that we have to configure it different than default

Reply:
Most users shouldn't need to change this.   However, if the user is using another app that also uses mark bits (or their own iptables rules), this allows our mark bit to be changed to avoid conflicts.